roles/tpa_single_node/templates/manifests/collectorist/api/Deployment.yaml.j2

```
@@ -0,0 +1,120 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: collectorist-api
+  namespace: "{{ tpa_single_node_namespace }}"
+  labels:
+    app.kubernetes.io/name: collectorist-api
+    app.kubernetes.io/component: collectorist
+    app.kubernetes.io/instance: redhat-trusted-profile-analyzer
+    app.kubernetes.io/version: 1.1.1
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/part-of: trusted-profile-analyzer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: collectorist-api
+      app.kubernetes.io/component: collectorist
+      app.kubernetes.io/instance: redhat-trusted-profile-analyzer
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: collectorist-api
+        app.kubernetes.io/component: collectorist
+        app.kubernetes.io/instance: redhat-trusted-profile-analyzer
+      annotations:
+        configHash/api: dac9d88322da8b671d9fc4f81500698ea05f108df8784b66db80d4242c9dee06
+        configHash/auth: d0dbc7f46adf81df142f3171f2234c47b7864a450fd1e404be4210c22caf81ea
+    spec:
+      volumes:
+        - name: tls
+          secret:
+            secretName: collectorist-api-tls
+        - name: config-auth
+          configMap:
+            name: collectorist-api-auth
+        - name: data
+          persistentVolumeClaim:
+            claimName: collectorist-api
+        - name: config
+          configMap:
+            name: collectorist-api
+      containers:
+        - name: service
+          image: "{{ tpa_single_node_trustification_image }}"
+          imagePullPolicy: IfNotPresent
+          command: ["/trust"]
+          args:
+            - collectorist
+            - api
+            - -p
+            - "8080"
+            - --csub-url
+            - http://guac-collectsub-pod:2782
+            - --auth-configuration
+            - /etc/config/auth.yaml
+            - --storage-base
+            - /data
+            - --collector-config
+            - /config/collectorist.yaml
+          env:
+# TODO
+#            - name: NAMESPACE
+#              valueFrom:
+#                fieldRef:
+#                  fieldPath: metadata.namespace
+            - name: RUST_LOG
+              value: info
+            - name: INFRASTRUCTURE_ENABLED
+              value: "true"
+            - name: INFRASTRUCTURE_BIND
+              value: "[::]:9010"
+            - name: HTTP_SERVER_BIND_ADDR
+              value: "::"
+            - name: HTTP_SERVER_TLS_ENABLED
+              value: "true"
+            - name: HTTP_SERVER_TLS_KEY_FILE
+              value: /etc/tls/tls.key
+            - name: HTTP_SERVER_TLS_CERTIFICATE_FILE
+              value: /etc/tls/tls.crt
+            - name: OIDC_PROVIDER_CLIENT_ID
+              value: "{{ tpa_single_node_oidc_provider_client_id }}"
+            - name: OIDC_PROVIDER_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  key: client_secret
+                  name: storage_secret
```

Please replace with following as we're now using oidc_secret :
```suggestion
            - name: OIDC_PROVIDER_CLIENT_ID
              valueFrom:
                secretKeyRef:
                  key: client_id
                  name: oidc_secret
            - name: OIDC_PROVIDER_CLIENT_SECRET
              valueFrom:
                secretKeyRef:
                  key: client_secret
                  name: oidc_secret
```

---

Done